--- reader.js.orig	2021-06-08 10:17:52.000000000 -0400
+++ reader.js	2021-06-08 13:47:16.000000000 -0400
@@ -4310,7 +4310,10 @@
 
 	$list.append(docfrag);
 	$list.find(".toc_link").on("click", function(event){
-			var url = this.getAttribute('href');
+            // convert href to be relative to the opf not ncx or nav
+            var ahref = this.getAttribute('href');
+            var basePath = book.packaging.navPath || book.packaging.ncxPath
+            var url = EPUBJS.core.resolveUrl(basePath, ahref)
 
 			event.preventDefault();
 
@@ -4344,4 +4347,4 @@
 	};
 };
 
-//# sourceMappingURL=reader.js.map
\ No newline at end of file
+//# sourceMappingURL=reader.js.map
